File: Backend/TrackrAPI/TrackrAPI/Models/AccesoAyuda.cs
using System;
using System.Collections.Generic;

namespace TrackrAPI.Models
{
    public partial class AccesoAyuda
    {
        public int IdAccesoAyuda { get; set; }
        public int IdAcceso { get; set; }
        public string? EtiquetaCampo { get; set; }
        public string DescripcionAyuda { get; set; } = null!;
        public byte[]? Imagen { get; set; }
        public string? TipoMime { get; set; }
        public int? Orden { get; set; }
        public string? NombreArchivo { get; set; }
        public int? IdAyudaSeccion { get; set; }

        public virtual Acceso IdAccesoNavigation { get; set; } = null!;
        public virtual AyudaSeccion? IdAyudaSeccionNavigation { get; set; }
    }
}

File: Frontend/trackr-administrador/src/app/shared/models/seguridad/acceso-ayuda.ts
export class AccesoAyuda {
  public idAccesoAyuda: number;
  public idAcceso: number;
  public etiquetaCampo: string;
  public descripcionAyuda: string;
  public orden: number;

  public imagen?: string;
  public tipoMime: string;

  // Extras para imagen
  public nombreArchivo: string;
  public imagenBase64: string;

  public idAyudaSeccion: number;
  public nombreAyudaSeccion: string;
  constructor() {}
}

export class AccesoAyudaImagenBase64{
  public idAccesoAyuda: number;
  public idAcceso: number;
  public etiquetaCampo: string;
  public descripcionAyuda: string;
  public orden: number;

  public imagen: string;
  public imagenBase64: string;
  public tipoMime: string;
  public nombreArchivo: string;

  constructor() {}
}

